File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomEmployeeRange/DbAtomEmployeeRangeEnum.cs
namespace DataModelLibrary.Database.AtomEmployeeRange;

/// <summary>資料庫-原子-人員規模-類型</summary>
public enum DbAtomEmployeeRangeEnum : short
{
    /// <summary>未選擇</summary>
    NotSelected = -1,

    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>不清楚</summary>
    Unclear = 1,

    /// <summary>少於100人</summary>
    LessThan100 = 2,

    /// <summary>100-500人</summary>
    Range100To500 = 3,

    /// <summary>500-1000人</summary>
    Range500To1000 = 4,

    /// <summary>1000人以上</summary>
    MoreThan1000 = 5
}

File: FrontEnd/manager-back-site/src/constants/DbAtomEmployeeRangeEnum.ts
/** 資料庫-原子-人員規模-類型 */
export enum DbAtomEmployeeRangeEnum {
 /** 未選擇 */
  NotSelected = -1,
  /** 未定義 */
 Undefined = 0,
 /** 不清楚 */
 Unclear = 1,
 /** 少於100人 */
 LessThan100 = 2,
 /** 100-500人 */
 Range100To500 = 3,
 /** 500-1000人 */
 Range500To1000 = 4,
 /** 1000人以上 */
 MoreThan1000 = 5
}